dataservice/api/family_relationship/models.py

```diff
@@ -0,0 +1,83 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+from dataservice.api.participant.models import Participant
+
+
+class FamilyRelationship(db.Model, Base):
+    """
+    Represents a relationship between two family members.
+
+    The relationship table represents an undirected graph. Only one
+    relationship will exist between any two participants.
+    (P1 -> P2 is equal to P2 -> P1 and only one of these will be stored)
+
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param participant_id: Kids first id of the first Participant in the
+    relationship
+    :param relative_id: Kids first id of the second Participant (or relative)
+    in the relationship
+    :param relationship_type: Text describing the nature of the
+    relationship (i.e. father, mother, sister, brother)
+    :param _rel_name: an autogenerated parameter used to ensure that the
+    relationships are not duplicated and the graph is undirected
+    """
+    __tablename__ = 'family_relationship'
+
+    _rel_name = db.Column(db.Text, unique=True)
+
+    participant_id = db.Column(
+        db.String(8),
+        db.ForeignKey('participant.kf_id'), nullable=False)
+
+    relative_id = db.Column(
+        db.String(8),
+        db.ForeignKey('participant.kf_id'), nullable=False)
+
+    relationship_type = db.Column(db.Text, nullable=False)
```

Comment:
No controlled vocab right now. I think in the future beyond MVP, we may have a separate role/relationship table with a controlled vocabulary.

Hmm what about `participant_to_relative_relation`? I like this because its one
field and provides clear directionality.